pkg/controller/controller_utils.go

```diff
@@ -28,6 +28,7 @@ import (
 	"time"
 
 	apps "k8s.io/api/apps/v1"
+	batch "k8s.io/api/batch/v1"
```

Comment:
I'm not sure if adding this dependency is appropriate. But I'll leave it to OWNERS.

Reply:
This is fine